eng/versioning/version_client.txt

```diff
@@ -178,7 +178,7 @@ com.azure.spring:azure-spring-boot-test-cosmosdb;1.0.0;1.0.0
 com.azure.spring:azure-spring-boot-test-keyvault;1.0.0;1.0.0
 com.azure.spring:azure-spring-boot-test-keyvault-reactive;1.0.0;1.0.0
 com.azure.spring:azure-spring-boot-test-parent;1.0.0;1.0.0
-com.azure.spring:azure-spring-boot-test-servicebus-jms;1.0.0;1.0.0
+com.azure.spring:azure-spring-cloud-test-servicebus-binder;1.0.0;1.0.0
```

Comment:
the azure-spring-boot-test-servicebus-jms is still needed to test our JMS starter.